src/DocumentFormat.OpenXml.Framework/Equality/OpenXmlElementEqualityOptions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace DocumentFormat.OpenXml.Equality
+{
+    /// <summary>
+    /// Options defining the behaviour of equality for <see cref="OpenXmlElement"/>.
+    /// </summary>
+    [Flags]
+    public enum OpenXmlElementEqualityOptions
```

Comment:
I think I'd prefer this to be a class in case we find we want more fine grained control